play.pokemonshowdown.com/src/battle-team-editor.tsx

```diff
@@ -925,6 +925,9 @@ class TeamTextbox extends preact.Component<{ editor: TeamEditorState, onChange?:
 					this.updateSearch();
 					this.setDirty = true;
 				}
+				if (selectionStart === 0 && selectionEnd === 0) {
+					this.clearInnerFocus();
+				}
```

Comment:
As far as I can tell, this would kick you out of inner focus if you cleared the input box while selecting your first pokemon (which, in addition to being a thing you might want to do normally, also happens if you select a search filter), which we don't want.